### ISSUE TITLE
Chisels and Bits incompatibility

### ISSUE THREAD

**Issue by SavageTheUnicorn** [iPortalTeam/ImmersivePortalsMod#1189]
As far as I know, these two mods are incompatible together, it's nothing as serious as a crash but any blocks 'chiseled' with the mod turns the block invisible immediately. After asking around in their discord server it's likely due to the way ImmPtl changes rendering. It could be a total game changer if compatibility was integrated and I'm sure the dev would be willing to work with you for it! Hell, I'd pay for compatibility purely so I can live in a tiny home. Please delete this if it's already been discussed elsewhere, I checked all active issues.

**Comment by SalTeyJek004** [comment]
compatability would be nice, but you can also use the pehkui mod on its own, or another mod that uses pehkui

**Comment by qouteall** [comment]
Have you tried bits and chisels?

**Comment by SavageTheUnicorn** [comment]
> Have you tried bits and chisels?

No, but I will now. I'm pretty sure it'll be a similar result though.

**Comment by SavageTheUnicorn** [comment]
> compatability would be nice, but you can also use the pehkui mod on its own, or another mod that uses pehkui

It has nothing to do with pehkui but the portals rendering changes themselves

**Comment by qouteall** [comment]
Someone said that bits and chisels is compatible but chisels and bits is incompatible so it may not be similar

**Comment by SavageTheUnicorn** [comment]
> Someone said that bits and chisels is compatible but chisels and bits is incompatible so it may not be similar

The fabric version is compatible but the forge versions are not, which is what I'm trying to use.
> Bits and Chisels (Fabric only)
>Chisels and Bits (Forge and Fabric)
